.changeset/late-monkeys-tease.md

```diff
@@ -0,0 +1,11 @@
+---
+'@graphql-yoga/apollo-managed-federation': minor
```

Comment:
isn't that an API change? for a library, it's a breaking changes, why not `major`?

Reply:
Because it is still v0.